### PR TITLE
Add info on Web apps & rosti; consolidate Domains table

### LINKED ISSUE #288: Zdokumentovat kdo má přístup na Roští
Pyvec má účet u Roští. Jede tam pyvo.cz, python.cz (?!), stránky PyConu a nový zpětnovazebník.
Já tam mám přístup a rád něco poměním, ale nevím kdo další má přístupy. Stálo by za to to zdokumentovat.

@honzajavorek, máš víc informací?

### PR DESCRIPTION
**Pull request by encukou**
- Add info on our account at rosti.cz (fixes #288). While doing that I somehow made other changes too:
- List our various Web apps
- Make the Domains table narrower, by moving some info out. Now it shouldn't scroll horizontally on desktop.

### CODE REVIEW

**Comment by encukou** [comment]
Hm, https://pyvec.com/ doesn't work :(

File: operations/web-apps.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+
++------------------------------------+------------------------+--------------------------------------+
+| Adresa                             | Web hosting            | Repozitář                            |
++====================================+========================+======================================+
+| https://python.cz                  | GitHub Pages           | :gh-repo:`pyvec/python.cz`           |
++------------------------------------+------------------------+--------------------------------------+
+| https://pyvec.org                  | GitHub Pages           | :gh-repo:`pyvec/pyvec.org`           |
++------------------------------------+------------------------+--------------------------------------+
+| http://pyvec.cz                    | Wedos redirect         | —                                    |
```
Comment (honzajavorek) [inline operations/web-apps.rst:16]: @sairon nefunguje redirect s https? 🤔

File: operations/web-apps.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
++------------------------------------+------------------------+--------------------------------------+
+| http://pyvec.com                   | Wedos redirect         | —                                    |
```
[the remark this answers is not in the document]
Comment (sairon) [reply operations/web-apps.rst:18]: Viz [komentář vedle](https://github.com/pyvec/docs.pyvec.org/pull/295/files#r1001510325).

File: operations/web-apps.rst [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
++------------------------------------+------------------------+--------------------------------------+
+| https://cz.pycon.org               | :ref:`rosti`           | :gh-repo:`pyvec/cz.pycon.org-2019`   |
++------------------------------------+------------------------+--------------------------------------+
+| https://pycon.cz                   | :ref:`rosti` ?         | —                                    |
```
[the remark this answers is not in the document]
Comment (sairon) [reply operations/web-apps.rst:22]: Nemám tam přístup, ale zjevně ten redirect dělá ten samý HTTP server, který hostuje samotnou Pycon prezentaci.

**Pull request merged**